src/NVika.Tests/Data/TestClass.txt

```diff
@@ -0,0 +1,11 @@
+﻿namespace InspectCodeTest
+{
+    public class TestClass
```

Comment:
Could you rename the file and class to something more meaningful, maybe like `Convert`?

Reply:
Have renamed to `Conversions`, hopefully that's a bit more fitting.

(Incidentally, when updating references, I've had a bit of trouble with the parser assuming that line endings are [always 1 character](https://github.com/laedit/vika/blob/ae90daa3ec1215181967d670a0f31e476d6e3f00/src/NVika/Parsers/InspectCodeParser.cs#L82-L83), which is not the case on Windows. This is probably not a huge issue because most git repositories are likely to be [configured to `core.autocrlf` correctly](https://git-scm.com/book/en/v2/Customizing-Git-Git-Configuration#_formatting_and_whitespace), and also out of scope of this pull, but I figure I may as well bring it up.)

Reply:
Thanks for the heads-up!